common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -285,9 +282,11 @@ def _can_user_view_content(self):
         Returns whether the runtime user can view the content
         of this sequential.
         """
+        course = self._get_course()
```

Comment:
Can you have the course be passed into this method, since the caller also needs to get the course?  This reduces a call to the modulestore.